components/automate-ui/src/app/entities/teams/team.reducer.ts

```diff
@@ -80,7 +80,7 @@ export function teamEntityReducer(state: TeamEntityState = TeamEntityInitialStat
       return set(
         'getStatus',
         EntityStatus.loadingSuccess,
-        teamEntityAdapter.addOne(action.payload, state)
+        teamEntityAdapter.upsertOne(action.payload, state)
```

Comment:
do we need upsert in the other success actions?

Reply:
Yeah I didn't wanna make such a sweeping change without tests though. Just did this to make the cypress tests I was writing work properly.

Reply:
Pretty sure I made a card to address this in general.

Reply:
https://github.com/chef/automate/issues/1199